dev/reference/sql/functions-and-operators/window-functions.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Window Functions
+summary: This document introduces window functions supported in TiDB.
+category: reference
+---
+
+# Window Functions
+
+The usage of window functions in TiDB is similar to that in MySQL. For details, see [MySQL Window Functions](https://dev.mysql.com/doc/refman/8.0/en/window-functions.html). 
```

Comment:
Just a small suggestion. It is similar to MySQL, but not the MySQL version TiDB is compatible with:
```suggestion
The usage of window functions in TiDB is similar to that in MySQL 8.0. For details, see [MySQL Window Functions](https://dev.mysql.com/doc/refman/8.0/en/window-functions.html). 
```